cmd/commands/root.go

```diff
@@ -84,5 +90,23 @@ func GetClientsets() *k8sutil.Clientsets {
 		logging.Fatal(err)
 	}
 
+	PreValidationCheck(ctx, clientsets, OperatorNamespace, CephClusterNamespace)
+
 	return clientsets
 }
+
+func PreValidationCheck(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorNamespace, cephClusterNamespace string) {
+	_, err := k8sclientset.Kube.CoreV1().Namespaces().Get(ctx, operatorNamespace, v1.GetOptions{})
+	if err != nil {
+		logging.Fatal(fmt.Errorf("Operator namespace '%s' does not exist. %v", operatorNamespace, err))
+	}
+	_, err = k8sclientset.Kube.CoreV1().Namespaces().Get(ctx, cephClusterNamespace, v1.GetOptions{})
+	if err != nil {
+		logging.Fatal(fmt.Errorf("CephCluster namespace '%s' does not exist. %v", cephClusterNamespace, err))
```

Comment:
Some of the commands may not require this to be fatal. For example, the rook version and updating an operator config setting doesn't require the ceph cluster to be running. And if the cluster namespace doesn't exist, the command will anyway fail later on something else. How about if we just log this error for the cluster namespace, but not exit? 
```suggestion
		logging.Error(fmt.Errorf("CephCluster namespace '%s' does not exist. %v", cephClusterNamespace, err))
```

Reply:
agreed.